### PR TITLE
DISPATCH-879 - Document how routers use alternate failover URLs

### PR DESCRIPTION
**Pull request by bhardesty**
@ganeshmurthy, can you please review for technical accuracy?

In the part where we talk about outgoing connections, I described a bit about how routers use the failover list. Also, in the part about managing outgoing connections, I added a command that can be used to view the failover list.

### CODE REVIEW

File: doc/new-book/configuration-connections.adoc [hunk 1]
```diff
@@ -56,7 +56,10 @@ If you have set up SSL/TLS or SASL in your environment, you can configure the ro
 [[adding_outgoing_connections]]
 == Adding Outgoing Connections
 
-Configuring outgoing connections involves setting the host and port on which the router should connect to other routers and brokers.
+Configuring outgoing connections involves setting the host and port on which the router connects to other routers and brokers.
+
+// Adding this here for now; in the future it might be better to have separate procedures for creating inter-router and route-container connections.
+When a router connects to a broker, the broker may provide backup connection data that the router can use if the primary connection fails. If the primary connection fails, the router attempts to reconnect by using a combination of the primary and -- if provided -- backup connections. For more information about viewing the backup connection data provided by the broker, see xref:managing_connectors[].
```
Comment (ganeshmurthy) [inline doc/new-book/configuration-connections.adoc]: Can we please add the following - 

If the router is unable to successfully connect using the primary or the backup connection information, it round robins around the list until it is able to make a successful connection.

File: doc/new-book/managing-using-qdmanage.adoc [hunk 2]
```diff
@@ -210,6 +211,13 @@ a|
 qdmanage query role port --type=connector
 ----
 
+|If the router is connected to a broker, view the alternate URLs on which the router can connect to the broker if the primary connection fails
+a|
+[options="nowrap",subs="+quotes"]
+----
+qdmanage query failoverList --name=_CONNECTOR_NAME_
```
Comment (ganeshmurthy) [inline doc/new-book/managing-using-qdmanage.adoc]: to be consistent the qdmanage query command should look like this - 

qdmanage query failoverList --type=connector --name=_CONNECTOR_NAME_

File: doc/new-book/configuration-connections.adoc (hunk 1 above)
Comment (bhardesty) [reply doc/new-book/configuration-connections.adoc]: Done.

File: doc/new-book/managing-using-qdmanage.adoc (hunk 2 above)
Comment (bhardesty) [reply doc/new-book/managing-using-qdmanage.adoc]: Done.

File: doc/new-book/configuration-connections.adoc [hunk 3]
```diff
@@ -56,7 +56,10 @@ If you have set up SSL/TLS or SASL in your environment, you can configure the ro
 [[adding_outgoing_connections]]
 == Adding Outgoing Connections
 
-Configuring outgoing connections involves setting the host and port on which the router should connect to other routers and brokers.
+Configuring outgoing connections involves setting the host and port on which the router connects to other routers and brokers.
+
+// Adding this here for now; in the future it might be better to have separate procedures for creating inter-router and route-container connections.
+When a router connects to a broker, the broker may provide backup connection data that the router can use if the primary connection fails. If the primary connection fails, the router attempts to reconnect by using a combination of the primary and -- if provided -- backup connections in round-robin fashion until the connection is successful. For more information about viewing the backup connection data provided by the broker, see xref:managing_connectors[].
```
Comment (jenmalloy) [inline doc/new-book/configuration-connections.adoc]: @bhardesty, Change "may" to "might"

**Pull request closed without merging**